contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,127 @@
+---
+date: 2022-08-29
+title: 'Array 1.39.0: The persons on events beta'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.39.0 introduces a new beta for you to try, a new display chart and big improvements to our client libraries.
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [guide to upgrading PostHog](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.39.0 release notes
+
+**Release highlights:**
+
+- Beta: [Persons on events](#beta-persons-on-events)
+- [New: Added features for client and server-side libraries](#new-added-features-for-client-and-server-side-libraries)
+- [New: Format your insights to show durations](#new-format-your-insights-to-show-durations)
+- [New: Big, bold number chart type](#new-big-bold-number-chart-type)
+- [Improved: Turbo properties](#improved-turbo-properties)
+- [Improved: History log now shows changes to insights](#improved-history-log-now-shows-changes-to-insights)
+- [Apps: Census integration](#apps-census-integration)
+
+### Beta: Persons on events
+
+We're getting ready to make a substantial change to the way [persons](/manual/persons) and [events](/manual/events) work by essentially combining them and adding person ids and properties _onto_ events. 1.39.0 gives you the chance to try the feature ahead of the full release. Here's how:
+
+- **PostHog Cloud:** No action is needed, as we're rolling this test out steadily behind a feature flag. If you're eager to be included from the start, let us know in [Slack](/slack) to see about joining our beta group.
+-  **Self-hosted users:** First, you need to [run an async migration](/docs/runbook/async-migrations). Afterwards, change your `actors_on_events_teams` [instance setting](/docs/self-host/configure/instance-settings) to `true`. As with other instance settings, you'll need to be [a Staff user](/docs/self-host/configure/instance-settings#staff-users) to make changes.
+
+Why are we doing this? Feedback showed that users weren't able to create queries based on person properties at the time of an event. By putting person properties on events, this becomes the new default, while still enabling you to to filter insights based on the latest properties using cohorts. 
```

Comment:
This is not the important thing here. Feedback from users has shown that query performance is paramount.